### PR TITLE
Add dictionary definition for 'bug'

### PR DESCRIPTION
**Pull request by shagunZ**
## This PR fixes...

The lack of dictionary definition and perspective for 'bug'

## What I did...

-Add dictionary definition for bug.
-Add dictionary perspective for bug.

## How to test...

_In a list format, tell us how to test this PR._
Ex:

1.Navigate to /dictionary
2.Search "Aesthetic"
3.See definition and perspective show up under term "aesthetic"
4.check grammar and types

## I learned...

As a beginner I learned how to contribute to open source repository.

### CODE REVIEW

**Comment by shagunZ** [comment]
kindly merge my pr adding hactoberfest label on it

File: src/assets/content/dictionary/bug.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 ---
 title: bug
-definition: ''
-perspectives: []
+definition: bug is a problem causing a program to crash or produce invalid output.
+sources:
+- sourceurl: https://www.techopedia.com/definition/24864/software-bug-
+perspectives:
+- meaning: bug can be an error, mistake, defect or fault, which may cause failure or deviation from expected results.
```
Comment (novellac) [inline src/assets/content/dictionary/bug.md:7]:
```suggestion
- meaning: an error, mistake, defect or fault, which may cause failure or deviation from expected results.
```
Suggestion: The strings will compile in the style of "As a <role>, <term> means <meaning>." so I've suggested to make the meaning just a fragment. You're welcome to accept the suggestion here in the Github UI, or to make an extra commit to the PR.

**Review by novellac — CHANGES_REQUESTED** [review]
I've suggested some changes to help the strings concat nicely. In addition, you may have to pull the most recent changes into your PR and fix a few merge conflicts.

**Comment by novellac** [comment]
Closing, as I've just merged in your previous for the same term, PR #178 .

**Pull request closed without merging**